### ISSUE TITLE
[BUG]API key fails to be confirmed with error: argument of type 'NoneType' is not iterable

### ISSUE THREAD

**Issue by Clint-G13** [CoinAlpha/hummingbot#2177]
**Describe the bug**
Updating to the latest development commit results to getting `argument of type 'NoneType' is not iterable` error when running `connect` command or starting a strategy.

**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):
1. Update bot to the latest development commit
2. Start bot
3. Run `connect` or import/create a strategy and start

**Screenshots**
![Connect - key not confirmed](https://user-images.githubusercontent.com/57189990/89558789-48919700-d847-11ea-9768-945def46bced.JPG)
![Binance docker](https://user-images.githubusercontent.com/57189990/89558795-4a5b5a80-d847-11ea-8cd7-0585deb43063.JPG)
Note: Coinbase and Ethereum wallet are not affected, and haven't tested Bittrex due to account issues.

**Release version**
0.30.0
```
"branch": "development",
"commit": "20cdbd70e3ed93d338f3156fe5b9e2e92c651c12",
"date": "2020-08-06T13:50:57Z",
```

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**

**Comment by TheHolyRoger** [comment]
Fixed this in https://github.com/CoinAlpha/hummingbot/pull/2176

**Comment by TheHolyRoger** [comment]
Well, I tried, they closed my PR.

**Comment by PtrckM** [comment]
confirmed fix on dev-0.30.0

![image](https://user-images.githubusercontent.com/54516858/89641364-184c0600-d8e4-11ea-9929-a0a5393527b6.png)
